assets/_scss/core_design.scss

```diff
@@ -78,6 +78,11 @@ p, li, dd {
     text-align: justify;
 }
 
+// Offset margin between paragraph and ul
+p+ul {
+    margin-top: -1rem;
```

Comment:
I very much disagree with this change. It makes more sense to me to keep the list indented from the surrounding text, keeping the vertical rhythm and adding a bit of air in between the paragraphs.

The left column below looks a bit more balanced to me personally.

![ul](https://user-images.githubusercontent.com/134321/121074038-9c586e00-c7d3-11eb-8501-f0d4222a3499.png)

I would even suggest a “counter-change”: Add the same space that's between paragraphs in between list items as well.

Reply:
I agree with Matej, not feeling strongly about the counter-change, though. If adding space between bullet items, I'd strongly lean towards a smaller spacing (maybe half of the spacing for paragraphs?).